recaf-ui/src/main/java/me/coley/recaf/ui/control/code/bytecode/AssemblerArea.java

```diff
@@ -106,10 +117,20 @@ public AssemblerArea(ProblemTracking problemTracking, AssemblerPipeline pipeline
 		pipeline.addBytecodeFailureListener(this);
 		pipeline.addBytecodeValidationListener(this);
 		pipeline.addPipelineCompletionListener(this);
+		pipeline.setDoUseAnalysis(false);
```

Comment:
Will eventually need to be turned back on _(ideally with optimizations via queue/cancellations discussed earlier today)_

Reply:
Optimizations discussed have been applied to `dev3` so we can remove this now.